src/lib/bcmp/bcmp_heartbeat.cpp

```diff
@@ -21,7 +18,7 @@ extern neighbor_discovery_callback_t
   \return ERR_OK if successful
 */
 BmErr bcmp_send_heartbeat(uint32_t lease_duration_s) {
-  BcmpHeartbeat heartbeat = {.time_since_boot_us = uptimeGetMicroSeconds(),
+  BcmpHeartbeat heartbeat = {.time_since_boot_us = bm_ticks_to_ms(bm_get_tick_count()),
```

Comment:
this one I technically time since boot microseconds.... not sure how hard that requirement is, but it is mentioned in the bcmp v1 google sheet. 

I'm up for using millis (maybe *100) for the time being and leaving a TODO for including a micro-seconds platform/bsp abstraction function.

Reply:
Agreed, oops meant to add it in there (*1000) but I had forgotten good catch.